wrappers/csharp/readme.md

```diff
@@ -1,6 +1,6 @@
 # .NET Wrapper for Intel RealSense SDK
 

```

Comment:
prerequisites:
 - .NET framework 3.5/4 (3.5 is required for Unity build)
- Visual Studio 2015/2017